src/seedu/addressbook/AddressBook.java

```diff
@@ -484,9 +497,12 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      */
     private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
+        for(String keyword: keywords){
+        	keyword.toLowerCase();
+        }
         for (String[] person : getAllPersonsInAddressBook()) {
-            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final ArrayList<String> wordsInNames=splitByWhitespace(getNameFromPerson(person).toLowerCase());
+            if (!Collections.disjoint(wordsInNames, keywords)) {
```

Comment:
Good work on making the match case-insensitive.

Remember to change the comment, help messages, tests etc to match this change.